### PR TITLE
Add gastracker; remove Commonwealth as their nodes are currently down

### PR DESCRIPTION
**Pull request by pyskell**
Preventative measure for increased traffic from the Coinbase addition. Will follow-up with more nodes.

### CODE REVIEW

**Comment by pyskell** [comment]
Added an ETC Cooperative run node as well.

**Comment by splix** [comment]
I also made a PR for GasTracker #2014

File: app/scripts/nodes.js [hunk 1]
```diff
@@ -99,8 +99,8 @@ nodes.nodeList = {
         'chainId': 61,
         'tokenList': require('./tokens/etcTokens.json'),
         'abiList': require('./abiDefinitions/etcAbi.json'),
-        'service': 'Ethereum Commonwealth',
-        'lib': new nodes.customNode('https://etc-geth.0xinfra.com', '')
+        'service': 'GasTracker (ETCDEV)',
```
Comment (splix) [inline app/scripts/nodes.js]: It's not ETCDEV project

Comment (pyskell) [reply app/scripts/nodes.js]: My mistake. Fixed.

**Review by gamalielhere — APPROVED** [review]

**Pull request merged**